src/scripts/components/Play.js

```diff
@@ -223,10 +233,25 @@ class Play extends Component<Props, State> {
         time: 0
       })
     }
+
+    const playerNode = document.querySelector(`#${PLAYER_ID}`)
+    if (playerNode) {
+      const parentNode: ?Node = playerNode.parentNode
+      if (parentNode) {
+        parentNode.removeChild(playerNode)
+      }
+    }
```

Comment:
☝️ this is a safeguard to try and finally remove the duplicate player problems. In case `clappr` hasn't cleaned up our `#player` node, we will.

Reply:
Yes, you are right. I think moving this to `componentWillUnmount` is probably a more sane for it to be.

Reply:
Changed via e7212e3